### PR TITLE
Add the basic skeleton and update travis

### PR DESCRIPTION
**Pull request by ZeroMagic**
- add the basic skeleton

- update Makefile

- update travis

- remove unused vendor and update vendor version

@resouer  PTAL.

Signed-off-by: ZeroMagic <anthonyliu@zju.edu.cn>

### CODE REVIEW

**Comment by resouer** [comment]
Thanks! This is huge. Could you please split this PR into several commits as below?

```
add the basic skeleton

update Makefile

update travis

remove unused vendor and update vendor version
```

**Comment by ZeroMagic** [comment]
Ok. This PR has been splited now.

File: cmd/containerd/containerd.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+	"github.com/containerd/containerd/cmd/containerd/command"
+	"github.com/sirupsen/logrus"
+	
+	_ "k8s.io/frakti"
+)
+
+func main() {
+	app := command.App()
+	logrus.Warn("This customized containerd is only for CI test, DO NOT use it for distribution.")
```
Comment (resouer) [inline cmd/containerd/containerd.go]: If this is for test, you should not vendor it, you can add a script to download and use standard containerd binary from its release.

Comment (ZeroMagic) [reply cmd/containerd/containerd.go]: With standard containerd binary, we can not use kata-runtime plugin and test it.

Comment (resouer) [reply cmd/containerd/containerd.go]: Then create a test dir for this at least.

Comment (ZeroMagic) [reply cmd/containerd/containerd.go]: Done it.

**Pull request merged**